### ISSUE TITLE
half-light radius is not circularized

### ISSUE THREAD

**Issue by moustakas** [legacysurvey/legacysurvey#134]
We should clarify that the `shape_r` half-light radii in the Tractor catalogs are along the semi-major axis; they are not circularized.

**Comment by weaverba137** [comment]
@moustakas the current description is
```rst
``shape_r``		              float32      arcsec                Half-light radius of galaxy model for galaxy type ``type`` (>0)
```
I propose changing this to
```rst
``shape_r``		              float32      arcsec                Half-light distance along semi-major axis of galaxy model for galaxy type ``type`` (>0)
```
As part of PR #136.

**Comment by weaverba137** [comment]
@moustakas could you please confirm that the description change in #136 looks good?

**Comment by moustakas** [comment]
Yes, that looks good, thanks.

**Comment by moustakas** [comment]
Linking to a notebook which @dstndstn wrote exploring this issue for posterity's sake--
  https://github.com/legacysurvey/legacypipe/blob/main/py/legacyanalysis/galaxy-ellipticities.ipynb

**Comment by weaverba137** [comment]
Closed by #136.
